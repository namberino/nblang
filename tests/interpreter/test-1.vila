in(2 * (3 / -"dummy"));
