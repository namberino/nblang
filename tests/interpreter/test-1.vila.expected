Phần tử phải là 1 số
Tại dòng 1
